Core/FitResult.hpp

```diff
@@ -19,6 +19,7 @@ namespace ComPWA {
 struct FitResult {
   FitParameterList InitialParameters;
   FitParameterList FinalParameters;
+  unsigned int NumFreeParameters;
```

Comment:
You mean that one should implement a function `numFreeParameter()`?

Reply:
Yeah, this thing is actually quite tricky. Normally it would be best to just pass the FitResult to the calculateAIC functions and extract it there. But the way you currently defined the AIC function makes more sense (than using FitResult), because it shows the user, that you should have a likelihood. So using a FitResult here would hide this information...
Therefore I was thinking of a namespace function like this `getFreeParameterCount(const FitResult&)`, which would also solve the problem. However `FitResult.NumFreeParrameters` is much easier to use.
I did not mark this part because I want it changed, but simply because I wanted to point this out and us to discuss which solution is best.